ansible/roles/installation/bpytop/files/bpytop.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Its better to use ansible modules then scripts and shell commands:
https://www.ansible.com/blog/ansible-best-practices-essentials

you can just use ansible snap modyle instead of having a shell script:
https://docs.ansible.com/ansible/latest/collections/community/general/snap_module.html#snap-module
https://stackoverflow.com/questions/47305658/install-snap-packages-with-ansible

For connecting, should be cleaner to just use ansible syntax instead of creating a separate sh file for such a short file, something like:

    - name: role name
      become: yes
      become_method: sudo
      shell: 'snap connect {{ item.app }}:{{ item.plug }}'
      with_items:
        - plug: password-manager-service
          app: nextcloud-client
        - plug: avahi-observe
          app: remmina